src/actions/sources/newSources.js

```diff
@@ -122,6 +123,20 @@ function checkPendingBreakpoints(sourceId) {
   };
 }
 
+function checkBlackBoxedSources(sources: Source[]) {
+  return async ({ dispatch }: ThunkArgs) => {
+    const tabs = prefs.tabsBlackBoxed;
```

Comment:
we should get the list of blackboxed tabs from the reducer

Reply:
> we should get the list of blackboxed tabs from the reducer

@jasonLaster updated, thanks